docs/packs_format.md

```diff
@@ -0,0 +1,59 @@
+---
+id: packs_format
+title: Packs Structure
+---
+
+As part of the content contribution process we are using a new format, Packs.
+This new content artifact will behave like a mini content repo. we will have all the relevant content items relevant 
+fot that pack located within it.
+<br/><br/>
+For instance a pack for CortexXDR will look as follows: 
+![](doc_imgs/67718809-342b8e80-f9d9-11e9-8a08-1b8ac87ffd77.png)
```

Comment:
I would remove the image. I don't want to update the image every time we change something small. Let's just provide a link to the directory in content repo.